harness/determined/common/api/_session.py

```diff
@@ -100,13 +158,102 @@ def put(
     ) -> requests.Response:
         return self._do_request("PUT", path, params, json, data, headers, timeout, False)
 
+
+class UnauthSession(BaseSession):
+    """
+    UnauthSession is mostly only useful to log in or unathenticated endpoints like /info.
+    """
+
+    def __init__(
+        self,
+        master: Optional[str],
+        cert: Optional[certs.Cert],
+        max_retries: Optional[urllib3.util.retry.Retry] = None,
```

Comment:
I've never liked the name `max_retries` for a `Retry` object in our code, even though I understand that `Retry(i)` creates a `Retry` object that says retry up to `i` times. Two questions:
1. Am I unaware of convention?
2. Should I file a ticket?

Reply:
No convention.  Probably was named by somebody who was familiar with our experiment config.

Don't file a ticket for internal renames.  You can ask me to rename it, or you can do it yourself.  But it's internal implementation and JIRA just doesn't really make sense for it.

Reply:
wikied